packages/frontend/src/components/chart/DesktopTokenControls.tsx

```diff
@@ -20,11 +20,14 @@ export function DesktopTokenControls({ tokens }: DesktopTokenControlsProps) {
   }
 
   return (
-    <div className="hidden h-full gap-6 md:flex">
+    <div
+      className="hidden h-full gap-6 md:flex"
+      data-role="chart-token-desktop-element"
+    >
       <div />
       <VerticalSeparator />
       <div className="flex flex-wrap items-baseline justify-start gap-x-4">
-        <span>View another token</span>
+        <span>View tokens</span>
         <div className="Dropdown">
```

Comment:
I've extracted the most I can into a new component named `TogglableDropdown`. I left the toggle button raw in a sense because I have currently no idea how to approach it. The issue is with the double button approach we're using for displaying the currently selected tokens. IMHO this should be an issue of a future refactor